### PR TITLE
Fix test cases init

### PR DESCRIPTION
**Pull request by anandjakhaniya**

### CODE REVIEW

File: backend/src/test/java/ai/verta/modeldb/TestsInit.java [hunk 1]
```diff
@@ -73,8 +73,11 @@ public class TestsInit {
   protected static HydratedServiceGrpc.HydratedServiceBlockingStub
       hydratedServiceBlockingStubClient2;
   protected static ArtifactStoreGrpc.ArtifactStoreBlockingStub artifactStoreBlockingStub;
+  private static Boolean setServerAndServiceRan = false;
 
   protected static void init() throws Exception {
+    if (setServerAndServiceRan) return;
```
Comment (jkwatson) [inline backend/src/test/java/ai/verta/modeldb/TestsInit.java]: style nit: I would greatly prefer we have brackets around all blocks

Comment (anandjakhaniya) [reply backend/src/test/java/ai/verta/modeldb/TestsInit.java]: Changed it.

File: backend/src/test/java/ai/verta/modeldb/TestsInit.java [hunk 2]
```diff
@@ -73,8 +73,13 @@ public class TestsInit {
   protected static HydratedServiceGrpc.HydratedServiceBlockingStub
       hydratedServiceBlockingStubClient2;
   protected static ArtifactStoreGrpc.ArtifactStoreBlockingStub artifactStoreBlockingStub;
+  private static Boolean setServerAndServiceRan = false;
```
Comment (jkwatson) [inline backend/src/test/java/ai/verta/modeldb/TestsInit.java]: I think this should be an AtomicBoolean with proper CAS access to make sure there aren't race conditions with the initialization.

Comment (anandjakhaniya) [reply backend/src/test/java/ai/verta/modeldb/TestsInit.java]: We are not executing test classes parallelly so race condition will never accrued. but lets make it for safety.

Comment (anandjakhaniya) [reply backend/src/test/java/ai/verta/modeldb/TestsInit.java]: Fixed it.

**Review by jkwatson-verta — APPROVED** [review]

File: backend/src/test/java/ai/verta/modeldb/TestsInit.java (hunk 2 above)
Comment (jkwatson-verta) [reply backend/src/test/java/ai/verta/modeldb/TestsInit.java]: The other reason would be cross-thread visibility. I don't think there's any guarantee that junit will run all the tests on the same thread, even if they aren't running concurrently. Thanks for the change! 👍

**Pull request merged**